pyqms/adaptors.py

```diff
@@ -255,7 +257,7 @@ def parse_evidence(
     if evidence_score_field is None:
         evidence_score_field = "PEP"  #  default
 
-    unimod_parser = pyqms.UnimodMapper()
+    unimod_parser = pyqms.UnimodMapper(xml_file_list=unimod_file_list)
```

Comment:
equally, unimod mapper has its own package and namespace so should not be imported into pyqms, I guess